docs/resources/aviatrix_smart_group.md

```diff
@@ -51,6 +51,14 @@ resource "aviatrix_smart_group" "test_smart_group_ip" {
       k8s_namespace  = "testnamespace"
       k8s_pod        = "testpod"
     }
+
+    match_expressions {
+      s2c = "remote-site-name"
```

Comment:
Can we add s2c and external as Optional arguments in the argument reference below as well?

Reply:
Also what is ext_args and does that need to be added as well?